8Knot/queries/query_template.py

```diff
@@ -1,26 +1,30 @@
 import logging
-import pandas as pd
-from db_manager.augur_manager import AugurManager
-from app import celery_app
-from cache_manager.cache_manager import CacheManager as cm
-import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
+import cache_manager.cache_facade as cf
 
 """
 TODO:
-(1) update QUERY_NAME
-(2) update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below.
-'NAME' should be the same as QUERY_NAME
-(3) paste SQL query in the query_string
-(4) insert any necessary df column name or format changed under the pandas column and format updates comment
-(5) reset df index if #4 is performed via "df = df.reset_index(drop=True)"
-(6) go to index/index_callbacks.py and import the NAME_query as a unqiue acronym and add it to the QUERIES list
-(7) delete this list when completed
+(1) Rename the query function to something informative. Replace other instances of "NAME_query" with your
+    chosen name.
+(2) Paste SQL query for Augur db into the query_string variable. Use the psycopg2 '%s' wildcard in the string
+    where you intend to use the repos list. The '%s' will be replaced with a literal representation of the 'repos' variable.
+(3) Navigate to 8Knot/pages/index/index_callbacks.py. Import your new query as a unique acronym. add it to the QUERIES list.
+    this registers your query to be scheduled when a user requests new data.
+(4) Create a table in 8Knot/8Knot/cache_manager/db_init.py for the new data you're retrieving from augur. Name the table identically to
+    your custom "NAME_query". e.g. if your query is "num_stars_query" the table should have the same name.
+(5) Update the docstring of the query to reflect the intention of the data being collected.
+(6) Delete this list when completed
+
+NOTE: Querying data from Augur is a Postgres->Postgres transaction. Any data transformations that will always
+    apply to visualization using the same data should either:
+
+    (a) be done in the SQL itself, so all records retrieved from Augur are usable or
+
+    (b) become analysis pre-processing steps that run on-the-fly on user request. Utilities for this kind of
+        processing are typically in 8Knot/8Knot/pages/utils/preprocessing_utils.py, but YMMV.
+
+        If you choose this path, PLEASE DOCUMENT this behavior.
 """
 
```

Comment:
All current visualizations are still using "QUERY_NAME" . We need to decide if we are still going to use that structure and keep that in the template or change all of the queries to be in consistent structure

Reply:
I'm seeing that all the queries use that variable for custom-naming the logs that are produced during collection. In the visualization template, that variable is just used to note that we're importing a query by its name. We can either remove the use of QUERY_NAME in the queries, or we can leave it as. I don't really mind either, because we're not using it in any APIs downstream.

Reply:
Yeah lets change it for it is consistent. Mainly for people to have a reference when making new queries and might get confused from the template vs the queries that exist.